### ISSUE TITLE
Where do you have failure enpoints?

### ISSUE THREAD

**Issue by veeramarni** [holotrek/ts-node-rest-api#4]
As you mentioned to redirect to https://github.com/holotrek/ts-node-rest-api/blob/master/src/api/middleware/facebook-auth.middleware.ts#L48 incase of oauth failure, is there an endpoint to add logic that url?

**Comment by holotrek** [comment]
It is assumed that you're using this with a SPA like Angular or React/Redux and the "clientAuthUrl" variable is the base URL for that app. I tend to use Angular CLI which is why I defaulted it to localhost:4200, which is the default port it uses, for development.

I'm working on some changes to this based on what I've found while attempting to use this in a real application. Once those are done, I will add a client SPA example.
